goinstant/goinstant.go

```diff
@@ -117,29 +124,31 @@ func composeUp(composeFile string) {
 	openBrowser("http://localhost:27517")
 }
 
-func composeDown(composeFile string) {
+func composeDown() {
+	home, err := os.UserHomeDir()
+	if err != nil {
+		fmt.Println("Error: ", err)
+	}
+
 	fmt.Println("Running on", runtime.GOOS)
 	switch runtime.GOOS {
 	case "linux", "darwin":
-		cmd := exec.Command("docker-compose", "-f", "-", "down")
-		cmd.Stdin = strings.NewReader(composeFile)
+		cmd := exec.Command("docker", "run", "--rm", "-v", "/var/run/docker.sock:/var/run/docker.sock", "-v", home+"/.kube/config:/root/.kube/config:ro", "-v", home+"/.minikube:/home/$USER/.minikube:ro", "--mount=type=volume,src=instant,dst=/instant", "--network", "host", "openhie/instant:test", "down", "-t", "docker")
 		cmd.Stdout = os.Stdout
 		cmd.Stderr = os.Stderr
 		err := cmd.Run()
 		if err != nil {
 			log.Fatalf("cmd.Run() failed with %s\n", err)
 		}
 	case "windows":
-		cmd := exec.Command("cmd", "/C", "docker-compose", "-f", "-", "down")
-		cmd.Stdin = strings.NewReader(composeFile)
+		cmd := exec.Command("cmd", "/C", "docker", "run", "--rm", "-v", "/var/run/docker.sock:/var/run/docker.sock", "-v", home+"\\.kube:/root/.kube/config:ro", "--mount=type=volume,src=instant,dst=/instant", "openhie/instant:test", "down", "-t", "docker")
```

Comment:
I see we make use of a `openhie/instant:test` image. Is the tag we are using (`test`) correct

Reply:
This should be latest, but that will only get built when this is in master.